src/core_plugins/kibana/public/context/api/anchor.js

```diff
@@ -56,7 +65,4 @@ function fetchAnchorProvider(courier, Private) {
   };
 }
 
-
-export {
-  fetchAnchorProvider,
-};
+export { fetchAnchorProvider };
```

Comment:
Why not just export the function inline?

Reply:
you're right, no idea why I did it this way :blush: